tools/diag_process_loopback.py

```
@@ -0,0 +1,173 @@
+    def run_on_mta(mode: str) -> None:
+        # Run on a FRESH thread so CoInitializeEx(MTA) succeeds (mirrors the app's
+        # dedicated capture thread). The main thread is already STA (PySide/comtypes).
+        def worker():
+            try:
+                comtypes.CoInitializeEx(comtypes.COINIT_MULTITHREADED)
+                print(f"[{mode}] worker CoInitializeEx(MTA) ok")
+            except OSError as exc:
+                print(f"[{mode}] worker CoInitializeEx(MTA) failed: {exc}")
+            try:
+                attempt(pid, not args.no_tree, mode)
+            except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)